app/views/admin/events/index.html.haml

```diff
@@ -176,14 +176,11 @@
                     method: :patch
 
             %td
-              - if event.state == "withdrawn"
-                Withdrawn
-              - else
-                .btn-group
-                  %button{:type=>"button", :class=>"btn btn-link dropdown-toggle", "data-toggle"=>"dropdown"}
-                    = event.state.humanize
-                    %span.caret
-                  %ul.dropdown-menu{:role=>"menu"}
-                    = render 'change_state_dropdown', event: event
+              .btn-group
+                %button{:type=>"button", :class=>"btn btn-link dropdown-toggle", "data-toggle"=>"dropdown"}
```

Comment:
Would be great if you can apply proper styling as well, ie new ruby syntax and single quotes.